lib/widgets/channel_colors.dart

```diff
@@ -6,137 +6,137 @@ import 'package:flutter_color_models/flutter_color_models.dart';
+  final ChannelColorSwatches b;
   final double t;
 
   @override
-  StreamColorSwatch _computeForBaseColor(int base) =>
-    StreamColorSwatch.lerp(a.forBaseColor(base), b.forBaseColor(base), t)!;
+  ChannelColorSwatch _computeForBaseColor(int base) =>
+    ChannelColorSwatch.lerp(a.forBaseColor(base), b.forBaseColor(base), t)!;
 }
 
 
-/// A [ColorSwatch] with colors related to a base stream color.
+/// A [ColorSwatch] with colors related to a base channel color.
 ///
 /// Use this in UI code for colors related to [Subscription.color],
 /// such as the background of an unread count badge.
-class StreamColorSwatch extends ColorSwatch<StreamColorVariant> {
-  StreamColorSwatch.light(int base) : this._(base, _computeLight(base));
-  StreamColorSwatch.dark(int base) : this._(base, _computeDark(base));
+class ChannelColorSwatch extends ColorSwatch<ChannelColorVariant> {
+  ChannelColorSwatch.light(int base) : this._(base, _computeLight(base));
+  ChannelColorSwatch.dark(int base) : this._(base, _computeDark(base));
 
-  const StreamColorSwatch._(int base, this._swatch) : super(base, _swatch);
+  const ChannelColorSwatch._(int base, this._swatch) : super(base, _swatch);
 
-  final Map<StreamColorVariant, Color> _swatch;
+  final Map<ChannelColorVariant, Color> _swatch;
 
   /// The [Subscription.color] int that the swatch is based on.
-  Color get base => this[StreamColorVariant.base]!;
+  Color get base => this[ChannelColorVariant.base]!;
 
-  Color get unreadCountBadgeBackground => this[StreamColorVariant.unreadCountBadgeBackground]!;
+  Color get unreadCountBadgeBackground => this[ChannelColorVariant.unreadCountBadgeBackground]!;
 
-  /// The stream icon on a plain-colored surface, such as white.
+  /// The channel icon on a plain-colored surface, such as white.
   ///
   /// For the icon on a [barBackground]-colored surface,
   /// use [iconOnBarBackground] instead.
-  Color get iconOnPlainBackground => this[StreamColorVariant.iconOnPlainBackground]!;
+  Color get iconOnPlainBackground => this[ChannelColorVariant.iconOnPlainBackground]!;
 
-  /// The stream icon on a [barBackground]-colored surface.
+  /// The channel icon on a [barBackground]-colored surface.
   ///
   /// For the icon on a plain surface, use [iconOnPlainBackground] instead.
   /// This color is chosen to enhance contrast with [barBackground]:
   ///   <https://github.com/zulip/zulip/pull/27485>
-  Color get iconOnBarBackground => this[StreamColorVariant.iconOnBarBackground]!;
+  Color get iconOnBarBackground => this[ChannelColorVariant.iconOnBarBackground]!;
 
-  /// The background color of a bar representing a stream, like a recipient bar.
+  /// The background color of a bar representing a channel, like a recipient bar.
   ///
-  /// Use this in the message list, the "Inbox" view, and the "Streams" view.
-  Color get barBackground => this[StreamColorVariant.barBackground]!;
+  /// Use this in the message list, the "Inbox" view, and the "Channels" view.
```

Comment:
Hmm, this line ideally belonged in the commit that renamed that view in the UI. Well, good to catch it now.

Reply:
Would it be better to have it in it's own NFC?